engine/schema/pom.xml

```diff
@@ -209,7 +209,7 @@
                                 <configuration>
                                     <checkSignature>true</checkSignature>
 <!--                                    <url>https://download.cloudstack.org/systemvm/${cs.version}/systemvmtemplate-${cs.version}.${patch.version}-xen.vhd.bz2</url>-->
-                                    <url>https://download.cloudstack.org/testing/systemvmtemplate/{cs.version}/systemvmtemplate-${cs.version}.${patch.version}-xen.vhd.bz2</url>
+                                    <url>https://download.cloudstack.org/testing/systemvmtemplate/${cs.version}/systemvmtemplate-${cs.version}.${patch.version}-xen.vhd.bz2</url>
```

Comment:
already in comment

Reply:
no @DaanHoogland they aren't the same, the one in the url will be the final path, where our official templates reside, since these templates are still under testing we've placed it in a testing path (where the nightly builds are uploaded)

Reply:
@Pearl1594 I wouldn't want to merge this unless one of the two lines is removed.

Reply:
Definitely @DaanHoogland . I didn't realize the PR was not a draft. I have converted it to a draft, once internal testing is done, I'll update the links and will then open it for merging.